app/models/search/tag_search_form.rb

```diff
@@ -7,14 +7,14 @@ class TagSearchForm
   ATTRIBUTES = [
     :query,
     :name,
-    :canonical,
```

Comment:
We should make sure if people keep URLs of current tag searches, those still work. Let's allow `canonical` here, and in `initialize` call a new method that converts `options[:canonical]` into the corresponding `options[:wrangling_status]`.